doc/releases/release-notes-3.7.rst

```diff
@@ -1761,6 +1762,30 @@ Libraries / Subsystems
 
 * POSIX API
 
+  * Improved Kconfig options to reflect standard POSIX Options and Option Groups.
+  * Added support for the following Option Groups
+    * ``POSIX_MAPPED_FILES``
```

Comment:
I suggest to directly link to the Kconfig options, as this also help folks easily get more info/context if they want

e.g

```suggestion
    * :kconfig:option:`CONFIG_POSIX_MAPPED_FILES`
```

Reply:
These are not Kconfig options. I could link to the relevant section in the option / option group documents though.

Reply:
Fixed!

Reply:
But support can be enabled using Kconfig options, no? I'm confused :) This link, for example, https://docs.zephyrproject.org/latest/kconfig.html#CONFIG_POSIX_MAPPED_FILES, is pretty much giving all the information I need re: this group **and** how to enable it?
If having "CONFIG_" appear in the text is a concern, it could be done as follows (didn't test it, but this should work):

```suggestion
    * :kconfig:option:`POSIX_MAPPED_FILES <CONFIG_POSIX_MAPPED_FILES>`
```

Reply:
> But support can be enabled using Kconfig options, no? I'm confused :)

They can.

The point of the release notes is to highlight what POSIX features have been added (and details), so it would make sense to point to the details of each added POSIX Option / Option Group.

> If having "CONFIG_" appear in the text is a concern

That's not really a concern.

I added an extra commit to #76114 that explicitly says which Kconfig option to use for each POSIX Option / Option Group